src/sandstorm/app-index/app-index.c++

```diff
@@ -163,7 +164,38 @@ public:
         httpResponse.setMimeType("application/octet-stream");
         httpResponse.initBody().setBytes(outStream.getArray());
 
-        return kj::READY_NOW;
+        if (!req.isSetState() ||
+            !response.getRoot<SubmissionStatus>().isPending()) {
+          return kj::READY_NOW;
+        }
+
+        // Send notification email to app index reviewers.
+        auto appTitle = indexer.getAppTitle(packageId);
+        auto notificationText = kj::str(
+            "An app package is pending review in the app index.\n\n"
+            "https://alpha.sandstorm.io/grain/NujwEZfut8oZoSdcrFzy9p/\n\n"
```

Comment:
I haven't seen any dupes but I think there was a case of multiple versions of an app being submitted in rapid succession.